### PR TITLE
Revert "[BEAM-5299] Define max timestamp for global window in proto (…

### PR DESCRIPTION
**Pull request by lgajowy**
…#6381)"

This reverts commit f20e90a65874945dbbeb33270cab2a91d619af03.

**Please** add a meaningful description for your change here

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

File: sdks/java/core/build.gradle [hunk 1]
```diff
@@ -51,8 +51,6 @@ test {
 }
 
 dependencies {
-  // Required to load constants from the model, e.g. max timestamp for global window
-  shadow project(path: ":beam-model-pipeline", configuration: "shadow")
```
Comment (mxm) [inline sdks/java/core/build.gradle]: Pretty sure this is the culprit.

**Comment by mxm** [comment]
@lgajowy and me are trying to find a fix but will revert otherwise.

File: sdks/java/core/build.gradle (hunk 1 above)
Comment (lgajowy) [reply sdks/java/core/build.gradle]: changed to compile, run the TextIOIT on Dataflow. Didn't help.

**Comment by lgajowy** [comment]
Run Java PreCommit

**Comment by lukecwik** [comment]
I don't think there is a good way to get this change in without breaking Dataflow temporarily. I believe the Dataflow container image needs to be rebuilt to include this change.

Note that this is slowly changing, more and more tests are migrating to use artifacts built from here instead of inside Google.

File: sdks/java/core/build.gradle (hunk 1 above)
Comment (lukecwik) [reply sdks/java/core/build.gradle]: I don't believe this to be the culprit.

The issue is that Dataflow has an internal container image that contains classes that were built from a previous source version. These classes become incompatible with versions submitted from "HEAD".

**Comment by lukecwik** [comment]
We could migrate the IOITs to build/run tests using Dataflow built from this github repo.

File: sdks/java/core/build.gradle (hunk 1 above)
Comment (mxm) [reply sdks/java/core/build.gradle]: Makes sense, the model project doesn't contain any of the conflicting classes. 

This should break quite often then, whenever classes are changed in `sdsks/java/core`.

**Comment by lukecwik** [comment]
Lets revert for now, we have two paths forward:
1) (BEST) Migrate Dataflow ITs to use Dataflow jar built at head
2) Update Dataflow worker image

**Comment by lgajowy** [comment]
I'm not sure 100% but it seems that nexmark_Dataflow jobs are affected too: 
https://builds.apache.org/view/A-D/view/Beam/job/beam_PostCommit_Java_Nexmark_Dataflow/873/consoleText

So: to not require rebuilding the dataflow container image we need to migrate both nexmark and IOITs. I don't have bandwidth to do this this week (taking 4 days off) so if no one else can do this - I suggest reverting.

**Comment by lukecwik** [comment]
@lgajowy Yes, I agree.

**Comment by mxm** [comment]
Let's trigger rebuilding of the image if it is not much work.

**Comment by lukecwik** [comment]
Its a good chunk of work to rebuild the image.

**Comment by mxm** [comment]
Is there a JIRA for 1)? Hasn't this already been done for other modules?

**Comment by lukecwik** [comment]
It has been done for several and it seems as though we missed nexmark and the perf test framework.

**Pull request merged**